src/pages/Trade/Trade.tsx

```diff
@@ -102,21 +111,58 @@ const TradePage: React.FC<RouteComponentProps> = (props) => {
         }
     };
 
+    const startTourForNewUser = localStorage.getItem(LOCAL_STORAGE_KEYS.NEW_USER_TOUR);
+
+    useEffect(() => {
+        if (startTourForNewUser == null && !isMobile) {
+            dispatch(setShowTour(true));
+        }
+    }, [dispatch, isMobile, startTourForNewUser]);
+
+    useEffect(() => {
+        if (showTour) {
+            document.documentElement.style.overflowX = 'inherit';
+            document.documentElement.style.scrollBehavior = 'inherit';
+        } else {
+            document.documentElement.style.overflowX = 'hidden';
+            document.documentElement.style.scrollBehavior = 'smooth';
+        }
+    }, [isMobile, showTour]);
```

Comment:
IsMobile is part of the dependency array but not used in the useEffect, is this by intent?

Reply:
Deleted.